src/main/java/org/jenkinsci/test/acceptance/junit/WithCredentials.java

```diff
@@ -43,6 +44,7 @@
 @Target({METHOD, TYPE})
 @Inherited
 @Documented
+@WithPlugins("credentials@2.0.7")
```

Comment:
Does this work on a rule? I thought it was only for actual tests. [reference](https://github.com/jenkinsci/acceptance-test-harness/blob/9e3ea2d23d825b1afd64bba5f359f763b64c4dd7/src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java#L92-L93)